### DISCUSSION TITLE
Internationalization for content as well as Admin UI

### DISCUSSION THREAD

**Discussion by TorbenHammes** [keystonejs/keystone#6670] [Feature requests]
Hey there,
it would be great if you could integrate the possibility to translate the dashboard as well as the possibility for multilingual content. Maybe there is something like that already, but I didn't find anything in the docs, unfortunately.

Greetings from Berlin
Torben

**Comment by vsava9** [comment]
Any info on this? At the moment what options do we have?

**Comment by Kwasula** [comment]
Lists have fields with label . How I can label  name of list in admin dashboard and in the list on the left side.  Now only way is to take content externaly and via graphql and name list for display there.
